web/src/app/services/auth.service.ts

```diff
@@ -42,14 +42,14 @@ export class AuthService {
       .signInWithPopup(new firebase.auth.TwitterAuthProvider()))
       .pipe(
         map((user) => this.formatUserResponse(user, 'twitter')),
-        catchError((error) => this.errorService.logError(error))
+        catchError((error) => this.errorService.logError(error)),
       );
   }
 
   signOutUser(): Observable<null> {
     return from(this.firebaseAuth.auth.signOut())
       .pipe(
-        catchError(err => this.errorService.logError(err))
+        catchError(err => this.errorService.logError(err)),
```

Comment:
Inconsistent with the catch above 

https://github.com/DashboardHub/TribeDashboard/pull/73/files#diff-8001f80fb85efec2dded95f6876dbce0R45